src/Main/Foedag.cpp

```diff
@@ -130,20 +130,19 @@ static std::filesystem::path GetProgramNameAbsolutePath(const char* progname) {
   return progname;  // Didn't find anything, return progname as-is.
 }
 
-void loadTclInitFile(CommandStack* commandStack,
-                     const std::string& initFilePrefix) {
+void loadTclInitFile(CommandStack* commandStack, ToolContext* context) {
   if (!commandStack) return;
 
   // Get the home and the local paths
   // std c++ doesn't have a concept of a home dir so we use QDir instead
   std::filesystem::path homeDir =
       std::filesystem::path(QDir::homePath().toStdString());
-  std::filesystem::path localDir =
-      std::filesystem::path(QDir::currentPath().toStdString());
-  std::vector<std::filesystem::path> searchPaths{homeDir, localDir};
+  std::filesystem::path localDir = std::filesystem::current_path();
+  auto etc = context->DataPath() / "etc";
```

Comment:
the *_init.tcl is different and it can be either in home dir or in cwd while the compilation flows.tcl must be in /etc/ folder of the install directory.